### PR TITLE
Fix for #4: Added ability to pass SwiftMailer log entries to `Yii::info()`

### PR DESCRIPTION
**Pull request by klimov-paul**
Fix for #4: Added ability to pass SwiftMailer log entries to `Yii::info()`

### CODE REVIEW

**Comment by klimov-paul** [comment]
@samdark, @germansokolov13, @slinstj, you are welcome for review

File: Logger.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\swiftmailer;
+
+use Yii;
+
+/**
+ * Logger is as SwiftMailer plugin, which allows passing of the SwiftMailer internal logs to the
```
Comment (samdark) [inline Logger.php]: is as → is a

File: Logger.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    {
+        Yii::info($entry, __METHOD__);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function clear()
+    {
+        // empty method stub
```
Comment (samdark) [inline Logger.php]: Should it stay empty? If yes, need a better comment i.e. "do nothing" so it's clear that method is actually "implemented".

**Comment by slinstj** [comment]
Excellent approach! Really good! I learned so much comparing your approach with mine!

A thing: what about the docs?

**Pull request merged**